tools/hmac/CalculateHmacPlatform.java

```diff
@@ -0,0 +1,53 @@
+import com.adyen.util.HMACValidator;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ Script to calculate the HMAC signature of Payments webhooks Banking/Management webhooks (where the signature is calculated
+ over the entire webhook payload)
+
+ Run with: mvn clean compile exec:java -Dexec.mainClass=CalculateHmacPlatform -Dexec.args= "{hmacKey} {path_to_JSON_file}"
+
+ cd tools/hmac
+ mvn clean compile exec:java -Dexec.mainClass=CalculateHmacPlatform -Dexec.args="11223344D785FBAE710E7F943F307971BB61B21281C98C9129B3D4018A57B2EB payload2.json"
+
+ */
+public class CalculateHmacPlatform {
+
+    public static void main(String[] args) {
+        if (args.length != 2) {
+            System.err.println("‼️Error running the script");
+            System.err.println("Usage: java CalculateHmacPlatform <hmacKey> <payloadFile>");
+            System.exit(1);
+        }
+
+        String hmacKey = args[0];
+        String jsonFilePath = args[1];
```

Comment:
nit: change suffix of filePath so that the user knows whether to specify its relative or absolute path (e.g. `jsonRelativePath` / `jsonAbsolutePath`)

Reply:
Path can be either relative (from where the script is run) or absolute.
The suggested approach (in README) is to update the content of the `payload.json` or `payload2.json` with the payload to test, then run the script as the README indicates.